### ISSUE TITLE
Error: channel closed

### ISSUE THREAD

**Issue by njam** [cargomedia/socket-redis#43]
Do we handle this correctly?
Connected to https://github.com/cargomedia/socket-redis/issues/42?

```
[2013-06-26 00:59:25.065] [ERROR] [default] - Uncaught exception: Error: channel closed
Error: channel closed
    at ChildProcess.target.send (child_process.js:397:26)
    at /usr/lib/node_modules/socket-redis/lib/server.js:190:11
    at Array.forEach (native)
    at module.exports.sendDownStatusRequest (/usr/lib/node_modules/socket-redis/lib/server.js:189:11)
    at Server.<anonymous> (/usr/lib/node_modules/socket-redis/lib/server.js:138:4)
    at Server.EventEmitter.emit (events.js:98:17)
    at HTTPParser.parser.onIncoming (http.js:2015:12)
    at HTTPParser.parserOnHeadersComplete [as onHeadersComplete] (http.js:119:23)
    at Socket.socket.ondata (http.js:1905:22)
    at TCP.onread (net.js:510:27)
```

**Comment by njam** [comment]
Waiting on https://github.com/cargomedia/puppet-cargomedia/issues/719

**Comment by tomaszdurka** [comment]
We don't have these errors in socket-redis logs anymore.
This bug is most likely fixed in #44.
Let's close this.
